configuration/sitemaps.md

```diff
@@ -537,16 +564,16 @@ Text item=Temperature valuecolor=[Last_Update=="Uninitialized"="gray",
                                   >=25="orange", >=15="green", 0="white", <15="blue"]
 ```
 
-In the first example, the `Weather` Item is colored blue (label and value) if the `Temperature` is below or equal to 4°C.
+In the first example, the `Weather` Item is blue (label and value) if the `Temperature` is at or below 4°C.
 
-Expressions will be evaluated from left to right.
-The first condition returning `true` will decide on the color.
-Looking at the second example, you will see, that the given five expressions are ordered in the only meaningful combination.
+The order in which expressions are listed is important.
+Note that expressions are evaluated from left to right, and the first matching expression determines the color.
+Looking at the second example, you will see that the order of the expressions is set so that the first match, left-to-right gives the correct result.
+If the order was reversed, color assignment would not work properly.
 
-`item_name` and `operator` are both optional.
-If not provided, the Item name will default to the current Item and operator will default to `==`.
-In the second example, this is shown by leaving out Temperature and by not giving a comparison operator in the expression `0="white"`.
-The following three lines are totally equal and valid:
+In the second example, the effect of omitting the `item_name` and `operator` parameters is shown by omitting `Temperature` and by omitting the comparison operator in the expression `0="white"` (as compared to `==0="white"`).
```

Comment:
Maybe not needed?

Reply:
Oh - I don't know...  I have not worked with colors in sitemaps before, and I found this second example helpful.  Up to you.  Let me know if you want to kill it.

Reply:
Let's leave it in, your rewording in the other comment should do the trick

Reply:
This comment has been addressed.